clients/tabby-agent/src/AgentConfig.ts

```diff
@@ -178,7 +178,7 @@ export const defaultAgentConfig: AgentConfig = {
   chat: {
     edit: {
       documentMaxChars: 3000,
-      commandMaxChars: 200,
+      commandMaxChars: 600,
```

Comment:
@icycodes i think it's safe to remove the command max chars check in agent, but leave the char clipping in individual command (e.g command trigger by user shall apply max chars clipping, but command triggered by features like "fix this" shall not having such clipping.

Reply:
The problem here is that the diagnostics should be counted as context rather than command characters. But it looks good to me to modify the maximum character limit in this PR.